### ISSUE TITLE
Dual Mining

### ISSUE THREAD

**Issue by neo4uo** [NebuTech/NBMiner#320]
Trying to setup dual mining at seperate pools and keep getting error 1

Please Help :(

`@ECHO OFF
nbminer -a ethash -do stratum+tcp://us1.ethermine.org:4444 -du 511aC0a3193d4A179C0D7921aC26277A88aa8Bdc.dagereAsus -d stratum+tcp://us-east.ezil.me:5555 -d zil1t054l2vw50mc2nhwwlscdhhrm7edr5d29eu6vq.dagereAsus -enable-dag-cache
pause `

Thanks

**Comment by pdieppa** [comment]
This is the proper format for mining ETH and ZIL.

nbminer -a ethash -o stratum+tcp://us-east.ezil.me:5555 -u 511aC0a3193d4A179C0D7921aC26277A88aa8Bdc.zil1t054l2vw50mc2nhwwlscdhhrm7edr5d29eu6vq.dagereAsus --enable-dag-cache

**Comment by diabaths** [comment]
in hiveos what is the format?

**Comment by pdieppa** [comment]
Create your wallet as a ETH and add your using the ETH.ZIL format.

Then create the flight sheet normally.

![image](https://user-images.githubusercontent.com/74556152/121075020-a6ae4400-c7a2-11eb-9834-b3dae902861f.png)

**Comment by diabaths** [comment]
and will farm in zill only when the pool open and the rest od the time will farm in hive pool??
also in "enable-dag-cache" i put what card i need to keep the dag??
like "enable-dag-cache" : "0,1" 

tnx for your help!

**Comment by pdieppa** [comment]
No, both examples I provided are for EZIL pool. Let me check if I have a flightsheet with a split pool.
As for enable-dag-cache, I don't think you have to specify anything.

**Comment by pdieppa** [comment]
![image](https://user-images.githubusercontent.com/74556152/121077258-68feea80-c7a5-11eb-9808-ffc0cdef7c3a.png)
The password is your zil-address@usw-eth.hiveon.net:4444

In your case, just the shardpool address with whatever zil pool you want to use.

**Comment by jonekluni** [comment]
but how to register ergo + zil? very necessary

**Comment by jonekluni** [comment]
hiveOS

**Comment by diabaths** [comment]
create a wallet your_eth_adreess.your_zil_adress

**Comment by twk4924123** [comment]
any idea how to setup for ergo+zil (ezil) on hiveos???

**Comment by pdieppa** [comment]
It' the same format as ETH, just change the correct pool for ergo and update your address to the ergo instead of eth.

**Comment by twk4924123** [comment]
i can mine eth+zil 
but ergo+zil wont work on nbminer.... anyone got working solution can share?? for hiveos please

**Comment by diabaths** [comment]
> 
> 
> i can mine eth+zil
> but ergo+zil wont work on nbminer.... anyone got working solution can share?? for hiveos please

can you show me your eth+zil flight??

**Comment by jonekluni** [comment]
> есть идеи как настроить ergo + zil (ezil) на hiveos ???


I really need it too

**Comment by browntownpr** [comment]
> > i can mine eth+zil
> > but ergo+zil wont work on nbminer.... anyone got working solution can share?? for hiveos please
> 
> can you show me your eth+zil flight??



> > i can mine eth+zil
> > but ergo+zil wont work on nbminer.... anyone got working solution can share?? for hiveos please
> 
> can you show me your eth+zil flight??

"ezil": "nbminer -a ethash -o stratum+tcp://us-east.ezil.me:5555 -u 0x1001ab2a86A3412bF55183e4687b8676cCeC0CEb.zil1lgv2avzu4mrhghe8ad8qu4mkwa7ldn7uk5de0f.hercules --enable-dag-cache"

**Comment by jonekluni** [comment]
> > > Я могу майнить eth + zil, 
> > > но ergo + zil не будет работать на nbminer .... у кого-нибудь есть рабочее решение, может поделиться ?? для улей, пожалуйста
> > 
> > 
> > можешь показать мне свой полет на eth + zil ??
> 
> > > Я могу майнить eth + zil, 
> > > но ergo + zil не будет работать на nbminer .... у кого-нибудь есть рабочее решение, может поделиться ?? для улей, пожалуйста
> > 
> > 
> > можешь показать мне свой полет на eth + zil ??
> 
> «эзил»: «nbminer -a ethash -o stratum + tcp: //us-east.ezil.me: 5555 -u 0x1001ab2a86A3412bF55183e4687b8676cCeC0CEb.zil1lgv2avzu4mrhghe8ad8qu4mkwa7dn-cacheldn-кеш

how to fit it into hiveos you can take a photo?

**Comment by jonekluni** [comment]
and how to register it in hiveOS ??? in the paragraph additionally it gives nothing. 
![Безымянный](https://user-images.githubusercontent.com/85689216/123473627-7d464280-d601-11eb-9825-64877f1822b0.jpg)

**Comment by jonekluni** [comment]
is there a solution for ergo + zil?

**Comment by budiusman** [comment]
> ![image](https://user-images.githubusercontent.com/74556152/121077258-68feea80-c7a5-11eb-9808-ffc0cdef7c3a.png)
> The password is your [zil-address@usw-eth.hiveon.net](mailto:zil-address@usw-eth.hiveon.net):4444
> 
> In your case, just change the shardpool address with whatever zil pool you want to use.

![image](https://user-images.githubusercontent.com/10285675/130183170-07476b19-1ea0-4303-9042-2cfdafc95dbe.png)
i'm mining ETH->2miner and ZIL-ezil , separate pool
are the setting in above image correct ? why i'm only see zil job from ezil but not eth job from 2miner
